coops-ui/src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -174,7 +174,8 @@ export default {
     },
 
     // Method to extract date from a local datetime string
-    formatDate (dateString) {
+    formatDate (dateString): string {
+      if (!dateString) return null // safety check
```

Comment:
I like what you're doing here! It might be a lot to add in with this PR because you would probably want to do all the functions but in our future refactor we could update this too `formateDate (dateString: string): string | null { .... }` to go heavy on the type'ing.

Reply:
I found this via unit test errors, but it's always good to write safe code 😛

Reply:
Absolutely!